README.md

```diff
@@ -116,6 +116,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://bugsnag.com/ - Free for up to 2000 errors a month after the initial trial
   * https://airbrake.io/ - Free for 1 project, 1 user, 2 errors per minute, 2 day retention
   * http://getsentry.com/ - Sentry tracks app exceptions in realtime, has a small free plan
+  * https://bugsplatsoftware.com/ - Free for 1 user, 1 project and 2000 reports per year
```

Comment:
nice, but not sure it fits the bill as software for devops/infradev types being a library to integrate into desktop software with a website to support it, thanks for the contrib but not gonna take this one sorry